frontend/src/types.ts

```diff
@@ -2,44 +2,59 @@
  * Common types, should be kept up to date with backend types
  */
 
-export type OdhApplication = {
-  metadata: {
-    name: string;
```

Comment:
name is required and should be defined here

Reply:
full schema for metadata is included using kpt typegen. See prior comment from me.